pyanaconda/ui/gui/spokes/subscription.py

```diff
@@ -34,6 +34,7 @@
 from pyanaconda.modules.common.constants.services import SUBSCRIPTION, NETWORK
 from pyanaconda.modules.common.structures.subscription import SystemPurposeData, \
     SubscriptionRequest
+from pyanaconda.modules.common.structures.secret import SECRET_TYPE_TEXT, SECRET_TYPE_HIDDEN
```

Comment:
These should be imported from `pyanaconda.core.constants`.

Reply:
Fixed.